addons/note/static/src/scss/note.scss

```diff
@@ -3,13 +3,18 @@
     text-decoration: line-through;
 }
 
-.o_note_form_view.o_form_view {
+.o_note_form_view {
     .o_form_sheet_bg {
         display: flex;
         flex-direction: column;
         padding: 0;
+        width: 100%;
+        max-width: 1200px;
```

Comment:
Why the `max-width` ?

Reply:
To keep it from resizing the chatter to be too small.